tests/integration/test_jdbc.py

```diff
@@ -0,0 +1,155 @@
+from time import sleep
+
+import pytest
+
+from dask_sql.server.app import _init_app, app
+from dask_sql import Context
+import pandas as pd
+from dask_sql.server.presto_jdbc import create_meta_data
+
+# needed for the testclient
+pytest.importorskip("requests")
+
+schema = "a_schema"
+table = "a_table"
+
+
+@pytest.fixture(scope="module")
+def c():
+    c = Context()
+    c.create_schema(schema)
+    row = create_table_row()
+    tables = pd.DataFrame().append(row, ignore_index=True)
+    tables = tables.astype({'AN_INT': 'int64'})
+    c.create_table(table, tables, schema_name=schema)
+
+    yield c
+
+    c.drop_schema(schema)
+
+
+@pytest.fixture(scope="module")
+def app_client(c):
+    c.sql("SELECT 1 + 1").compute()
+    _init_app(app, c)
+    # late import for the importskip
+    from fastapi.testclient import TestClient
+
+    yield TestClient(app)
+
+    app.client.close()
+
+
+def test_jdbc_has_schema(app_client, c):
+    create_meta_data(c)
+
+    check_data(app_client)
+
+    response = app_client.post("/v1/statement", data="SELECT * from system.jdbc.schemas")
+    assert response.status_code == 200
+    result = get_result_or_error(app_client, response)
+
+    assert "columns" in result
+    assert "data" in result
+    assert "error" not in result
+    assert result["columns"] == [
+        {
+            "name": "TABLE_CATALOG",
+            "type": "varchar",
+            "typeSignature": {"rawType": "varchar", "arguments": []},
+        },
+        {
+            "name": "TABLE_SCHEM",
+            "type": "varchar",
+            "typeSignature": {"rawType": "varchar", "arguments": []},
+        }
+    ]
+    assert len(result["data"]) == 3
+    assert result["data"] == [
+        ["", "root"],
+        ["", "a_schema"],
+        ["", "system_jdbc"],
+    ]
```

Comment:
Wondering if there is a way to place these assertions in a helper function like `assert_resp` to simplify the tests; okay if this isn't possible